website/pages/en/index.js

```diff
@@ -153,6 +154,7 @@ const Grammar = props => (
       {
         content: '<div class="typeset">Create structured clauses and contracts by binding variables in natural language to a data model. Optionally, clauses can be made executable by adding contract logic to the data model.</div>',
         image: imgUrl('grammar.png'),
+        imageAlt: "Natural Language templates",
```

Comment:
Any reason why the different cases (`L` or `l` / `t` or `T` ?). Maybe use `Template` everywhere.